lib/manager-webpack4/src/manager-config.ts

```diff
@@ -42,8 +42,26 @@ export const getAutoRefs = async (
 };
 
 const checkRef = (url: string) =>
-  fetch(`${url}/iframe.html`).then(
-    ({ ok }) => ok,
+  fetch(`${url}/iframe.html`, {
+    headers: {
+      Accept: 'application/json',
+    },
+  }).then(
+    async ({ ok }) => {
+      if (ok) {
+        // so the status is ok, but if we'd ask for JSON we might get a response saying we need to authenticate.
+        const data = await fetch(`${url}/iframe.html`, {
+          headers: {
+            Accept: 'application/json',
+          },
+        });
```

Comment:
Why do we refetch the same URL with the same headers again?

Reply:
Ow it's supposed to be with different headers!

Reply:
I want to check with default headers first, and make a secondary check with JSON headers, because the act of having the accept headers might make it fail?